samples/forseti/forseti_conf.yaml.sample

```diff
@@ -0,0 +1,92 @@
+# Copyright 2017 Google Inc.
```

Comment:
There should be only one config file being merged in here.  I have moved the previous ```forseti_conf.yaml``` to this one with a ```.sample``` suffix.  What do you think?